src/commands/command.ts

```diff
@@ -1,7 +1,19 @@
-import {ArgDescriptor} from 'command-line-args';
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
 
+import {ArgDescriptor} from 'command-line-args';
 import {ProjectConfig} from 'polymer-project-config';
-export {ProjectConfig} from 'polymer-project-config';
```

Comment:
I don't think this is a particularly bad export. It's fine to export something that you assert is part of your public contract, and in fact insulates the client from changes somewhat, say in the case that commands extended the config interface somehow.